src/actions.js

```diff
@@ -64,10 +64,11 @@ const actions = {
       content: text,
     }
   },
-  changeCellType(cellType) {
+  changeCellType(cellType, language = undefined) {
```

Comment:
should the default language just be `js` here? Seems like this is the place to show the default.

Reply:
duh yes of course! good call, thanks! :-)